jest.config.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
-  modulePathIgnorePatterns: ["<rootDir>/examples", "<rootDir>/.tmp"],
+  modulePathIgnorePatterns: [
```

Comment:
[This guy](https://github.com/facebook/jest/issues/8114#issuecomment-475068766) seems to think that using `modulePathIgnorePatterns` will work. It doesn't in our case? It's just a shame because if it did we could omit the rename script and job altogether.

Reply:
well we have that config in place already 😅, so if it did we wouldn't be here 😭